frontend/packages/topology/src/components/list-view/cells/AlertsCell.tsx

```diff
@@ -60,6 +62,16 @@ const AlertsCell: React.FC<AlertsProps> = ({ item }) => {
   const { podData, loaded } = usePodsWatcher(resource);
   const { buildConfigs, loaded: buildConfigsLoaded } = useBuildConfigsWatcher(resource);
   const { loaded: rcsLoaded, mostRecentRC } = useReplicationControllersWatcher(resource);
+  const workloadRqAlert = useResourceQuotaAlert(item);
+  // const workloadRqAlertVariant = (workloadRqAlert?.variant as NodeStatus) || NodeStatus.default;
```

Comment:
Hi @divyanshiGupta ,

nit, can you remove this comment?

Since #12029 is merged can you rebase the PR?

Reply:
@lokanandaprabhu done.